src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocstyle/bothfiles/package-info.java

```diff
@@ -13,4 +13,16 @@
 
 */
 
-package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocstyle.bothfiles; // violation
+/** Example */ // violation 'First sentence should end with a period.'
+@Example
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocstyle.bothfiles;
```

Comment:
Why we changing input ?
It was only only one line, no comment, but it becomes a way different

Reply:
Reverted and added new test.